api/src/test/java/org/openmrs/module/fhir2/api/impl/FhirEncounterServiceImplTest.java

```diff
@@ -404,38 +405,68 @@ public void searchForEncounter_shouldReturnCollectionOfEncounterBySubject() {
 		    new SearchQueryBundleProvider<>(theParams, dao, encounterTranslator, globalPropertyService, searchQueryInclude));
 		when(searchQueryInclude.getIncludedResources(any(), any())).thenReturn(Collections.emptySet());
 		
-		IBundleProvider results = encounterService.searchForEncounters(null, null, null, subject, null, null, null, null);
+		IBundleProvider results = encounterService.searchForEncounters(null, null, null, subject, null, null, null, null,
+		    null);
 		
 		List<IBaseResource> resultList = get(results);
 		
 		assertThat(results, Matchers.notNullValue());
 		assertThat(resultList, not(empty()));
 		assertThat(resultList, hasSize(greaterThanOrEqualTo(1)));
 	}
-	
+
 	@Test
 	public void searchForEncounter_shouldReturnCollectionOfEncounterByUUID() {
+		List<Encounter> encounters = new ArrayList<>();
+
 		TokenAndListParam uuid = new TokenAndListParam().addAnd(new TokenParam(ENCOUNTER_UUID));
-		
+
 		SearchParameterMap theParams = new SearchParameterMap().addParameter(FhirConstants.COMMON_SEARCH_HANDLER,
-		    FhirConstants.ID_PROPERTY, uuid);
-		
+				FhirConstants.ID_PROPERTY, uuid);
+
 		when(dao.getSearchResults(any(), any())).thenReturn(Collections.singletonList(openMrsEncounter));
 		when(dao.getSearchResultUuids(any())).thenReturn(Collections.singletonList(ENCOUNTER_UUID));
 		when(encounterTranslator.toFhirResource(openMrsEncounter)).thenReturn(fhirEncounter);
 		when(searchQuery.getQueryResults(any(), any(), any(), any())).thenReturn(
-		    new SearchQueryBundleProvider<>(theParams, dao, encounterTranslator, globalPropertyService, searchQueryInclude));
+				new SearchQueryBundleProvider<>(theParams, dao, encounterTranslator, globalPropertyService, searchQueryInclude));
```

Comment:
I will reedit this,  my IDE is set up to match OpenMRS style guide, but most times I prefer pushing code from vs-code so maybe it distorts code formart.

Reply:
well you could search more on how to avoid automatic edits in the ide you are using

Reply:
@jnsereko We actually have pretty aggressive code formatting functionality in this module. Just run `mvn clean package -DskipTests` and look for any changes (though please only commit changes to files you've already touched for some reason). This will get things formatted correctly.